schema/features-schema.js

```diff
@@ -17,7 +17,10 @@ const schema = Joi.object().keys({
                         Joi.object().keys({
                             contextName: Joi.string().required(),
                             operator: Joi.string().required(),
-                            values: Joi.array().items(Joi.string())
+                            values: Joi.array().items(Joi.string()),
+                            caseInsensitive: Joi.bool().optional(),
+                            inverted: Joi.bool().optional(),
+                            value: Joi.alternatives().try(Joi.string(), Joi.number()).optional()
```

Comment:
@chriswk will it cause problems in java if we allow the value property to be either a number or a string?

Reply:
Sorry for the delayed answer, but yes, I don't think Gson knows how to handle a type incertainity. This will require a major version, because it is a backwards incompatible change to go from a guaranteed string to either/or

Reply:
make sense. We could have seperate fields per type, aka `dateValue`, `strValue` and so on, but feels a bit verbose. I vote for keeping it as a string as well.

Reply:
I agree with keeping it a string, could be changed to number in next major, seems pointless to do a major release due to such small change. The SDKs could parse the string to correct format easily based on the operator.

The `dateValue`, `strValue` etc. indeed does sound too verbose.

Reply:
We use just `value` when the operator only uses a single value, and it will always be a string, requiring the SDK to parse it per operator.

Reply:
fixed!